### DISCUSSION TITLE
Send all entered responses to participants (e.g. via email)

### DISCUSSION THREAD

**Discussion by nicebread** [rubenarslan/formr.org#514] [Q&A]
Hi all,

some survey platforms have the option (at the end of the survey): "Send me an email with all the data I entered".

I understand that all entered data can be accessed as an data frame, so it should be possible in principle. I just wonder if anybody already implemented that.

Cheers,
Felix

**Comment by rubenarslan** [comment]
HI Felix, this could be more convenient. By default, formr only delivers the items you mention explicitly. So, my approach for this involves putting all item names in a comment and then printing the data frame.

```
# survey_name item1 item2 item3 ... item20
kable(survey_name)
```

Best wishes,

Ruben
